.github/workflows/deploy-to-playstore.yml

```diff
@@ -35,12 +35,12 @@ jobs:
           keyStorePassword: ${{ secrets.KEYSTORE_PASSWORD }}
           keyPassword: ${{ secrets.KEY_PASSWORD }}
 
-      - name: Deploy to Production
-        uses: r0adkll/upload-google-play@v1.0.19
+      - name: Deploy to Alpha
+        uses: r0adkll/upload-google-play@v1.0.0
```

Comment:
@misshannah You should have bumped this up to `v1.1.1`,  not downgrade it to `v1.0.0`

Reply:
@jumaallan I assumed we're starting over. It's a new app separate from the one we did in 2022.

Reply:
Actually, thanks for explaining, I'll update it to v1.1.1